README.md

```diff
@@ -49,4 +43,11 @@ $ ./bin/estserver \
     -ca-key <pkidir>/local-ca.key \
 ```
 
+TODO: add details on the API usage by means of command line utilities (openssl, curl, etc).
+For example, how to succeed with this call
+```
+curl -X POST --cert client.pem --key pkey.pem --cacert root.crt https://msul-ota:8443/.well-known/est/simplereenroll
+Invalid content-type: . Must be application/pkcs10The CSR could not be decoded: asn1: syntax error: sequence truncated
+```
```

Comment:
It would be great to provide a step by step guide on a device certificate rotation by using command line utilities against the server.